MatchGame/Deck.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MatchGame
+{
+    public class Deck : IDeck
+    {
+        private List<Card> _cards;
+
+        public Deck()
+        {
+            _cards = BuildStandardPack();
+        }
+
+        public List<Card> Cards => _cards;
```

Comment:
This should have been part of the interface, and thus mockable for testing in the Game class.